dap/BESStoredDapResultCache.cc

```diff
@@ -247,24 +235,19 @@ BESStoredDapResultCache *
 BESStoredDapResultCache::get_instance()
 {
     if (d_enabled && d_instance == 0) {
-        d_instance = new BESStoredDapResultCache();
+        static BESStoredDapResultCache instance;
         d_enabled = d_instance->cache_enabled();
         if(!d_enabled){
-            delete d_instance;
-            d_instance = NULL;
             BESDEBUG("cache", "BESStoredDapResultCache::"<<__func__ << "() - " <<
                 "Cache is DISABLED"<< endl);
+            return nullptr;
        }
         else {
```

Comment:
You can remove this else {} and practically you can drop the BESDebug lines

Reply:
10-4